lib/views.js

```diff
@@ -384,19 +384,15 @@ exports.tasks_pending = {
      *  message needs the `to` and `message` properties
      */
     function hasPending(tasks) {
-      var tasks = tasks || [];
-
-      tasks.forEach(function(task) {
+      return tasks && tasks.some(function(task) {
         if (task && task.state === 'pending') {
-          task.messages.forEach(function(msg) {
+          return task.messages && task.messages.some(function(msg) {
             if (msg && msg.to && msg.message) {
               return true;
```

Comment:
It happened, in the sense that the code works (as noted by the tests that I wrote, which I wrote against the master version of this view, which broke when I tried them against the code that Gareth points out as broken, and which work against this code.

IMO `some` is the most correct thing to do here, but it can be interpreted clearly as a style thing.